takwimu/templates/profile/profile_detail_takwimu.html

```diff
@@ -1,6 +1,22 @@
 {% extends 'profile/profile_detail_base.html' %}
 {% load staticfiles sass_tags geotags %}
 
+{% block profile_header_tail %}
+<div class="column-full">
+    <strong>{{ primary_releases.active.name }} {{ primary_releases.active.year }}</strong>
+    {% if primary_releases.other %}
+    <div class="tool-group toggle-sub-group release-list">
+        <a>Change release<i class="fa fa-chevron-circle-down"></i></a>
+        <ul class="sub-group" style="display: none;">
+        {% for release in primary_releases.other %}
```

Comment:
Still this shows the wrong release year here.

![screenshot from 2018-12-13 14-11-53](https://user-images.githubusercontent.com/1779590/49935838-35e48480-fee3-11e8-968d-5374100ee473.png)

Tanzania census was not in **2017**